### PR TITLE
 CA-276606: xapi-inventory port to jbuilder and xcp-inventory compatibility package

### PR DESCRIPTION
**Pull request by kc284**
Signed-off-by: Konstantina Chremmou <konstantina.chremmou@citrix.com>

### CODE REVIEW

**Comment by mseri** [comment]
Please retitle the commit as: 
```
CA-276606: xapi-inventory port to jbuilder and xcp-inventory compatibility package
```

File: packages/xs/xcp-inventory.1.0.2/url [hunk 1]
```diff
@@ -0,0 +1 @@
+archive: "https://github.com/xapi-project/xcp-inventory/archive/v1.0.2.tar.gz"
```
Comment (mseri) [inline packages/xs/xcp-inventory.1.0.2/url]: Please add the checksum (it's just the `md5sum` of the `v1.0.2.tar.gz` file)

File: packages/xs/xapi-inventory.1.1.0/url [hunk 2]
```diff
@@ -1 +1,2 @@
-archive: "https://github.com/xapi-project/xcp-inventory/archive/v1.0.2.tar.gz"
+archive: "https://github.com/xapi-project/xcp-inventory/archive/v1.1.0.tar.gz"
+checksum: 6476ec8c14dcc426394ac2dc4d7bb67b
```
Comment (mseri) [inline packages/xs/xapi-inventory.1.1.0/url]: Missing quotes. It should be:
```
checksum: "6476ec8c14dcc426394ac2dc4d7bb67b"
```

File: packages/xs/xcp-inventory.1.1.0/url [hunk 3]
```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/xapi-project/xcp-inventory/archive/v1.1.0.tar.gz"
+checksum: 6476ec8c14dcc426394ac2dc4d7bb67b
```
Comment (mseri) [inline packages/xs/xcp-inventory.1.1.0/url]: As above

Comment (kc284) [reply packages/xs/xcp-inventory.1.1.0/url]: 😳

**Comment by mseri** [comment]
This is currently failing due to a missing dependency in a `xcp-idl` `jbuild` file. Should be fixed by https://github.com/xapi-project/xcp-idl/pull/183 

One the fix is merged I'll retrigger the build

**Comment by mseri** [comment]
Currently failing due to https://github.com/xapi-project/xen-api/pull/3366

**Pull request merged**